fighters/link/src/acmd/tilts.rs

```diff
@@ -65,25 +54,20 @@ unsafe fn attack_lw3(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
     frame(lua_state, 1.0);
-    if is_excute(fighter) {
-        FT_MOTION_RATE(fighter, 0.769);
-    }
+    FT_MOTION_RATE(fighter, 8.0/(11.0-1.0));
```

Comment:
This doesn't equate to 0.769, the new motion rating makes the startup look awkward

Reply:
Changed FT_MOTION_RATE(fighter, 8.0/(11.0-1.0)); to:
FT_MOTION_RATE(fighter, 7.69/(11.0-1.0));